samples/highcharts/blog/rotating-3d-scatter/demo.js

```diff
@@ -9,7 +9,7 @@ Highcharts.getOptions().colors = Highcharts.getOptions().colors.map(
             },
             stops: [
                 [0, color],
-                [1, Highcharts.Color(color).brighten(-0.2).get('rgb')]
+                [1, new Highcharts.Color(color).brighten(-0.2).get('rgb')]
```

Comment:
So this hack is no longer working?

https://github.com/highcharts/highcharts/blob/f39c2d61d63b5ef36eebdcdd5985eea193790bf8/ts/Core/Color/Color.ts#L133-L136

If so, it is ok. But then we can remove those lines.

Reply:
Yes, it is no longer working. It is not possible to call class constructor without new in ES6. Error: `Uncaught TypeError: Class constructor Color cannot be invoked without ‘new’`. I will remove the old hack.